File: src/main/java/com/dh/ondot/member/core/exception/AppleUserParseFailedException.java
package com.dh.ondot.member.core.exception;

import com.dh.ondot.core.exception.NotFoundException;

import static com.dh.ondot.core.exception.ErrorCode.APPLE_USER_PARSE_FAILED;

public class AppleUserParseFailedException extends NotFoundException {
    public AppleUserParseFailedException() {
        super(APPLE_USER_PARSE_FAILED.getMessage());
    }

    @Override
    public String getErrorCode() {
        return APPLE_USER_PARSE_FAILED.name();
    }
}

File: src/main/kotlin/com/dh/ondot/member/core/exception/AppleUserParseFailedException.kt
package com.dh.ondot.member.core.exception

import com.dh.ondot.core.exception.ErrorCode.APPLE_USER_PARSE_FAILED
import com.dh.ondot.core.exception.NotFoundException

class AppleUserParseFailedException :
    NotFoundException(APPLE_USER_PARSE_FAILED.message) {
    override val errorCode: String get() = APPLE_USER_PARSE_FAILED.name
}
